server/src/test/java/com/netcracker/MessageTest.java

```diff
@@ -0,0 +1,32 @@
+package com.netcracker;
+
+import com.netcracker.domain.Message;
+import org.junit.Test;
+
+import static com.netcracker.domain.MessageType.NAME_REQUEST;
+import static com.netcracker.domain.MessageType.TEXT;
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+public class MessageTest {
+    @Test
+    public void shouldReturnMessageTypeWhenAskingForIt() {
+        Message message = new Message(TEXT, "Message text");
+
+        assertEquals(message.getMessageType(), TEXT);
+    }
+
+    @Test
+    public void shouldReturnTextWhenAskingForIt() {
```

Comment:
&#x1F34E; **Issue**
Your test method name could convey the meaning more precisely. Namely, it should return **data** when asking for it.